app/controllers/loginController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+	
+	class LoginController{
+
+		public function LoginController(){
```

Comment:
This is old, PHP4 style constructor. Use __construct
Functions with class name will not be treated as contructors as of PHP 5.3.3. Details: http://php.net/manual/en/language.oop5.decon.php